Makefile

```diff
@@ -183,12 +184,20 @@ test: vendor $(ADDON_ASSET_FILE)
 .PHONY: integration
 integration: $(MINISHIFT_BINARY)
 	mkdir -p $(INTEGRATION_TEST_DIR)
+<<<<<<< 611b4898ed2fbfb034b7926fb1b33998c9ffc732
 	go test -timeout $(TIMEOUT) $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) -tags basic $(GODOG_OPTS)
+=======
+	go test -timeout $(TIMEOUT) $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) --exec-before-feature="$(EXEC_BEFORE_FEATURE)" $(GODOG_OPTS)
```

Comment:
@agajdosi Looks like you added rebase conflict here please rebase again and resolve conflicts.

Reply:
@praveenkumar Missed that, will try to fix it. Thanks.